libvast/vast/format/zeek.hpp

```diff
@@ -258,6 +260,8 @@ class reader final : public single_layout_reader {
   vast::schema schema_;
   type type_;
   record_type layout_;
+  std::ptrdiff_t proto_field_;
+  std::vector<size_t> port_fields_;
```

Comment:
Why are the port fields `size_t` and the proto field `ptrdiff_t`?

Reply:
because I used the value -1 to indicate that there is no `proto` field. I can switch to `optional<size_t>` if you insist.

Reply:
switched